pkg/monitor/api.go

```diff
@@ -33,40 +33,45 @@ const (
 	LocatorOAuthAPIServerReusedConnection     = "oauth-apiserver-reused-connection"
 )
 
-// Start begins monitoring the cluster referenced by the default kube configuration until
-// context is finished.
-func Start(ctx context.Context) (*Monitor, error) {
-	m := NewMonitorWithInterval(time.Second)
+func GetMonitorRESTConfig() (*rest.Config, error) {
```

Comment:
I fail to understand why you modify Start() and extract this function, is something related to the openshift tests structure?

Reply:
> I fail to understand why you modify Start() and extract this function, is something related to the openshift tests structure?

Yeah.  trying to keep the clients consistent and "obvious" in the flow.